chore: add drop-checker page

preschian opened the pull request:
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Related: https://github.com/kodadot/ops-internal/issues/1768

Demo: https://deploy-preview-10578--koda-canary.netlify.app/drop-checker/ahp/158

![Screenshot 2024-07-11 at 16 59 28](https://github.com/kodadot/nft-gallery/assets/734428/7eda9145-80e8-4290-a980-676e4aaac181)

vikiival commented:
Scr?

Jarsen136 reviewed (approved):
looks cool

hassnian reviewed (commented):
nice!

just a thought,  what if we added ❌ or ✅ if the onchain and database data was `=== `  similar to what we do in the code-cheker

![CleanShot 2024-07-13 at 12 26 23@2x](https://github.com/user-attachments/assets/75b4297a-da52-491c-9167-c85ce5200498)

small darkmoda stuff, otherwise LGTM

![CleanShot 2024-07-13 at 12 17 22@2x](https://github.com/user-attachments/assets/4b2452d8-5100-4eb5-be7a-747ce3f65b3d)

can't see the text here 
![CleanShot 2024-07-13 at 12 14 36@2x](https://github.com/user-attachments/assets/572d154b-08ed-44e6-8e7a-6b46f952d9d6)

exezbcz commented:
looks good, where can i access the page?

vikiival commented:
> looks good, where can i access the page?

> Demo: https://deploy-preview-10578--koda-canary.netlify.app/drop-checker/ahp/158

so `<url>/drop-checker/<prefix>/<id>`

preschian commented:
> just a thought, what if we added ❌ or ✅ if the onchain and database data was `===` similar to what we do in the code-cheker

added 👍🏻



> small darkmoda stuff
> can't see the text here

fixed

Jarsen136 commented on pages/drop-checker/[chain]/[collection-id].vue:
```diff
@@ -0,0 +1,308 @@
+const { chainSymbol, decimals } = useChain()
+
+// get data from database
+const data = await getDrops({
+  chain: [chain],
+  limit: 10,
+})
+const drops = data.filter((drop) => drop.collection === collectionId)[0]
```
It only gets information from the first 10 drops. Could we make it 100 or so?

preschian replied:
oh, correct, let's use 100 for now
later on, we need to fetch drop by chain & collection id: https://github.com/kodadot/private-workers/issues/200
